assets/js/googlesitekit/datastore/forms/index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * core/forms Data store: values
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Data from 'googlesitekit-data';
+import invariant from 'invariant';
+import { STORE_NAME } from './constants';
+
+export { STORE_NAME };
+
+const SET_FORM_VALUES = 'SET_FORM_VALUES';
+
+export const INITIAL_STATE = {};
+
+export const actions = {
+	setValues( formName, formData ) {
+		invariant( formName, 'formName is required for setting values.' );
+		invariant( formData !== undefined, 'formData is required.' );
```

Comment:
Technically we want `formData` to be an object, so the `!== undefined` check is overly verbose. Checking for truthiness is fine.